wine/8/Dockerfile

```diff
@@ -0,0 +1,35 @@
+## use latest build at base that still has wine 8 (8.0.2)
+FROM            ghcr.io/parkervcp/yolks:wine_latest@sha256:db82ec2e67fe90d7d66500c3de64a079051a34ad7d2a5ed480c8c72117f9af04

```

Comment:
not really the best option, since wine 8 is eol. They should start reporting issues to Wine Bugtracker

Reply:
For some games like palworld the latest image an conan wine 9 will refuse console output or cause crashes